spec/services/get_directions_service_spec.rb

```diff
@@ -14,8 +14,13 @@
   let(:store_coordinates) { [106.66796536861601, 10.794422140319528] }
```

Comment:
should use simple, "dummy value" here like 106, 10 to prevent unnecessary memory storage or precision error